Calculator/ViewController.swift

```diff
@@ -22,15 +24,62 @@ class ViewController: UIViewController {
         // Dispose of any resources that can be recreated.
     }
 
-    @IBAction func didgitButtonTapped(_ sender: Any) {
-        let button = sender as! UIButton
-        let buttonText = button.titleLabel!.text!
-        if resultArea.text! == "0" {
-            resultArea.text! = buttonText
+    @IBAction func operationButtonTapped(_ sender: Any) {
+        guard let button = sender as? UIButton else {
+            return
+        }
+        
+        calc.inputValue = Int(resultArea.text!)!
+        switch button.titleLabel!.text! {
+        case "+":
+            calc.nextOperation = .plus
+        case "-":
+            calc.nextOperation = .minus
+        case "x":
+            calc.nextOperation = .multiply
+        case "/":
+                calc.nextOperation = .divide
+        default:
+            fatalError("Something gone wrong with =")
```

Comment:
We need to remove this fatal error case. At least we can just ignore wrong operations.